src/modules/peek/Peek.FilePreviewer/Controls/DriveControl.xaml

```diff
@@ -68,29 +62,25 @@
         <Grid
             Grid.Row="1"
             Grid.ColumnSpan="2"
-            RowSpacing="5">
+            RowSpacing="4">
             <Grid.RowDefinitions>
                 <RowDefinition Height="auto" />
                 <RowDefinition Height="auto" />
             </Grid.RowDefinitions>
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="auto" />
-                <ColumnDefinition Width="*" />
-            </Grid.ColumnDefinitions>
             <Border
                 x:Name="CapacityBar"
                 Grid.Row="0"
                 Grid.ColumnSpan="2"
-                Height="20"
+                Height="24"
                 Background="{ThemeResource AccentFillColorDisabledBrush}"
-                CornerRadius="10" />
+                CornerRadius="12" />
 
             <Border
                 Grid.Row="0"
                 Grid.ColumnSpan="2"
-                Height="20"
+                Height="24"
```

Comment:
This looks pretty big in the screenshot.. maybe 16 is better?

Reply:
I think is a case of an opinion.

Reply:
W11 Settings seems to use 16/18px (probably a mismatch with the 1px border).

So yeah, I do think the 16px height is the better value here.

![image](https://github.com/microsoft/PowerToys/assets/9866362/c74457cd-07e6-4639-b87b-f90ee5988f76)